### DISCUSSION TITLE
Is UxPlay affected by the screen mirroring block in EU?

### DISCUSSION THREAD

**Discussion by thiccaxe** [FDH2/UxPlay#355] [Q&A]
title.

**Comment by fduncanh** [comment]
This is in Apples hands.    It seems to be  a new  feature to control an iphone from a mac, so obviously UxPlay will not control an iphone.


https://www.heise.de/en/news/Apple-optimizes-iPhone-mirroring-on-the-Mac-but-not-in-the-EU-9958882.html
